Makefile

```diff
@@ -17,33 +17,33 @@ help:
 
 .PHONY: dev
 dev: node_modules/.uptodate
-	tox -e py35-dev
+	tox -e py36-dev
 
 .PHONY: lint
-lint:
-	tox -e py35-lint
+lint: node_modules/.uptodate
```

Comment:
Make `make lint` automatically install the JavaScript dependencies if they're missing or out of date